BlueBrain/nexus-web
# 1020 disable sorting when a text search is applied

Fixes BlueBrain/nexus#1020

### How to test

- Open a Project view
- Search for a resource in the List
- Notice, that Sort button is disabled
- When you clear a Search, the Sort button should reappear
